modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/enricher/ContentParameterModelEnricher.java

```diff
@@ -85,8 +85,10 @@ private void configureDsl(ParameterDeclaration p) {
   }
 
   private void defaultToPayload(ParameterDeclaration p) {
-    p.setRequired(false);
-    p.setDefaultValue(PAYLOAD);
+    if (p.getDefaultValue() == null) {
+      p.setRequired(false);
```

Comment:
is this because the `required` is going to be handled by the @Optional enricher?